### ISSUE TITLE
Refactor Filtering examples to use Data (part 1)

### ISSUE THREAD

**Issue by JCoder121** [grommet/hpe-design-system#3108]
## Acceptance Criteria

- [ ] FilteringCards, FilteringLists, and FilteringRangeSelector examples updated to use Data and related components

Context: the Data refactor ticket was broken up into individual tickets per example. Refactor Filtering examples to use Data component: https://design-system.hpe.design/templates/filtering?q=filter

**Comment by JCoder121** [comment]
2/3/23 update: 
- Currently refactoring first two examples, the goal is to completely finish refactoring functionality, then extrapolate results to remaining 3 examples

**Comment by JCoder121** [comment]
2/10/23 update:

All examples have been refactored and are in the first round of review. Connecting with Eric and Taylor on finer details and design direction within some of the examples.
Remaining work: final rounds of review + iteration.

**Comment by JCoder121** [comment]
2/14/23 remaining work:

- Filterservers example: blocked on nested object filter definition issue (see https://github.com/grommet/grommet/issues/6638)

- Filteringrangeselector example: refactor using Page and Pagecontent

- Filteringselect example: blocked on inline filter issue (see https://github.com/grommet/hpe-design-system/issues/3150), refactor using Page and Pagecontent

**Comment by JCoder121** [comment]
2/21/23: 
Split this issue into what is currently ready for final review, still linked to this PR. New issue created for FilterServers and FilteringSelect, which are both blocked

**Comment by JCoder121** [comment]
3/3/23

Ticket is under final review and iteration and should be merged today or early next week
